rc/filetype/rust.kak

```diff
@@ -39,10 +39,9 @@ add-highlighter shared/rust/line_doctest region ^\h*//[!/]\h*```($|should_panic|
 add-highlighter shared/rust/line_doctest/marker region ```.* $ group
 add-highlighter shared/rust/line_doctest/marker/fence regex ``` 0:meta
 add-highlighter shared/rust/line_doctest/marker/keywords regex [\d\w] 0:meta # already matched above, just ignore comma
-add-highlighter shared/rust/line_doctest/inner region '^\h*//[!/]\h+#' ' ' group
+add-highlighter shared/rust/line_doctest/inner region '^\h*//[!/]( #(?= )|)' ' ' group
```

Comment:
I guess `( #(?= ))?` would be more idiomatic.

I'm wondering, what's special about `#` followed by a space in doc comments? I thought only `#!` has special meaning and everything else should be treated the same way.
If that's actually true, then something like this would look more correct to me:
```diff
diff --git a/rc/filetype/rust.kak b/rc/filetype/rust.kak
index 8a57ce9e..881eeeec 100644
--- a/rc/filetype/rust.kak
+++ b/rc/filetype/rust.kak
@@ -42 +42 @@ add-highlighter shared/rust/line_doctest/marker/keywords regex [\d\w] 0:meta # a
-add-highlighter shared/rust/line_doctest/inner region '^\h*//[!/]( #(?= )|)' ' ' group
+add-highlighter shared/rust/line_doctest/inner region '^\h*//[!/]' ' ' group
@@ -58 +58 @@ add-highlighter shared/rust/block_doc/doctest/inner default-region group
-add-highlighter shared/rust/block_doc/doctest/inner/hidden regex '^\h*\**\h*#' 0:meta
+add-highlighter shared/rust/block_doc/doctest/inner/hidden regex '^\h*\**\h*' 0:meta
```

Reply:
A `#` inside a doctest is used for text that's part of the test but not part of the documentation: https://doc.rust-lang.org/rustdoc/documentation-tests.html#hiding-portions-of-the-example

Reply:
Oh that's nifty, thanks!